### DISCUSSION TITLE
Can I access the registry from my test?

### DISCUSSION THREAD

**Discussion by ronsig** [redhat-developer/vscode-extension-tester#1585] [Q&A]
I've tried using 'winreg-ts' to access the registry from within my test. No error is thrown, but no value is returned either.
same for the os version
Is there something preventing access to the registry? Is a workaround needed?

Thanks

**Comment by djelinek** [comment]
Hello, there is not known limitations for using another packages to access anything 🙂 

Tip: eg for the mentioned getting of OS version, take a look on `node:os` https://nodejs.org/api/os.html
